### ISSUE TITLE
Make code managable

### ISSUE THREAD

**Issue by szepeviktor** [aimeos/map#13]
Hello!

Were you thinking about splitting up the `Map` class into traits the way methods are categorized in README?
Thank you!

**Comment by aimeos** [comment]
Yes, traits might be a good idea to spit the code into several files

**Comment by szepeviktor** [comment]
Proposal for remaining parts.

- properties
- constructor
- implementation methods for `\ArrayAccess, \Countable, \IteratorAggregate, \JsonSerializable`
- protected functions

**Comment by aimeos** [comment]
Dividing the source code like that looks reasonable.
Can you create a PR that splits the code like that?

**Comment by szepeviktor** [comment]
No, I'm sorry. I work with code very slowly 🐌
